Assets/Scripts/EnemyController.cs

```diff
@@ -16,130 +18,120 @@ public class EnemyController : MonoBehaviour
     public int maxSpeed;
     public int shieldRegen;
     public float weight;
-    public string movementType;
+    public MovementType movementType;
     public Vector3 movementTarget;
     public List<Transform> path; // hierarchy objects should be ordered
     public int currentPathTarget = 0;
     public float distanceTolerance;
-
+    public float speed = 0.05f;
 
     private Rigidbody2D enemyRigidBody;
+
+    // START Unity magic functions
     void OnDeath()
     {
-        duration = Time.time-spawnTime;
-        Debug.Log("A " + type + " enemy died after " + duration+ " seconds!");
+        duration = Time.time - spawnTime;
+        Debug.Log(string.Format("A {0} enemy died after {1} seconds!", type, duration));
     }
 
     void OnHurt()
     {
 
     }
-    // Start is called before the first frame update
+
+    // Awake is called before the first frame update
     void Awake()
     {
-        // initialize iD
-           // get enemyCounter from level, increment by 1, set this to that.
+        // TODO: initialize id
```

Comment:
If you have items you know you need to do later, putting TODO at the beginning of the comment allows your editor (usually Visual Studio for Windows, or XCode for Mac) to put it in a task list window so you can easily check what your previous self wanted your present or future self to do.